astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -1,15 +1,37 @@
-from datetime import timedelta
+import os
+from datetime import datetime, timedelta
 
-from airflow.models.dag import DAG
-from airflow.utils.timezone import datetime
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.s3 import (
+    S3CreateBucketOperator,
+    S3DeleteBucketOperator,
+)
+from airflow.providers.amazon.aws.transfers.local_to_s3 import (
+    LocalFilesystemToS3Operator,
+)
 
-from astronomer.providers.amazon.aws.sensors.s3 import S3KeySensorAsync
+from astronomer.providers.amazon.aws.sensors.s3 import (
+    S3KeySensorAsync,
+    S3KeySizeSensorAsync,
+    S3KeysUnchangedSensorAsync,
+)
 
 default_args = {
     "retry": 5,
     "retry_delay": timedelta(minutes=1),
 }
 
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "astronomer-airflow-providers")
```

Comment:
PROJECT_ID is referring to GCP, correct it please

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/86/commits/93d4a9ec4f724b29a84a50c66fe1c1b51347cfe8)